catalog/from-k8s/syncer.go

```diff
@@ -62,6 +62,9 @@ type ConsulSyncer struct {
 	// ConsulK8STag is the tag value for services registered.
 	ConsulK8STag string
 
+	//K8sConsulPrefix prepends K8s services in Consul with a prefix
+	K8sConsulPrefix string
```

Comment:
Is this additional struct field necessary? It doesn't seem to be used anywhere in the syncer.